apps/nuxt3-ssr/components/LogoMobile.vue

```diff
@@ -16,8 +17,14 @@ if (logoFileName) {
 <template>
   <NuxtLink :to="link" class="block">
     <span class="sr-only">Go to home</span>
-    <i v-if="svg" v-html="svg"></i>
-
+    <img
+      v-if="image"
+      :src="image"
+      width="200"
+      height="50"
+      style="background-color: white"
+    />
```

Comment:
use tailwind for themable style

Reply:
bg-white?

Reply:
sure , what about images with white foreground ?